how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -0,0 +1,175 @@
+# How to use Kata Containers and "cri" (contaienrd plugin)
```

Comment:
- I think it should be `CRI`, not `"cri"` (throughout the doc).
- s/contaienrd/containerd/